src/plugins/eth/ponder.config.ts

```diff
@@ -57,7 +61,7 @@ export const config = createConfig({
       network: "mainnet",
       abi: RESOLVER_ABI,
       address: factory({
-        address: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e",
+        address: REGISTRY_ADDRESS,
```

Comment:
I've checked the `linea.eth` and the `base.eth` plugins, and each of them uses the registry address for the resolver contract factory method. Which matches what you did here as well 👍